migration/migration/008-install-uda.sql

```diff
@@ -5,6 +5,14 @@ AS $func$
 $func$
 LANGUAGE SQL STABLE PARALLEL SAFE;
 
+CREATE OR REPLACE FUNCTION _prom_catalog.get_timescale_minor_version()
```

Comment:
they were added in the idempotent section but since idempotent runs last in the migration sequence, you cannot always rely on the functions existing early enough in the fresh install to use them on new migrations.